packages/eslint-plugin/configs/es5.js

```diff
@@ -73,30 +73,6 @@ module.exports = {
 				'!': true,
 			},
 		} ],
-		'valid-jsdoc': [ 'error', {
-			prefer: {
-				arg: 'param',
-				argument: 'param',
-				extends: 'augments',
-				returns: 'return',
-			},
-			preferType: {
-				array: 'Array',
-				bool: 'boolean',
-				Boolean: 'boolean',
-				float: 'number',
-				Float: 'number',
-				int: 'number',
-				integer: 'number',
-				Integer: 'number',
-				Number: 'number',
-				object: 'Object',
-				String: 'string',
-				Void: 'void',
-			},
-			requireParamDescription: false,
-			requireReturn: false,
-		} ],
 		'valid-typeof': 'error',
```

Comment:
Does it mean that `es5` config will no longer validate JSDoc comments?

Reply:
Correct. If that's the intention also, you'd have to have es5.js `extend` jsdoc.js (and then probably remove jsdoc.js from the top-level, since it already imports es5.js there).

Good catch.

Reply:
Based on the previous implementation and knowing that sometimes we want to explicitly use ES5 I guess you are all correct.